site/sfguides/src/hex_titanic/titanic.md

```diff
@@ -0,0 +1,192 @@
+author: aefendic@hex.tech and chase.romano@snowflake.com
+id: end_to_end_model_development_with_snowpark_and_hex
+summary: This lab will walk you through how to use Snowflake and Hex.
+categories: data-science-&-ml,partner-integrations
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Hex, Notebooks, Partner Connect
+
+# End to End Model Development with Hex and Snowpark
+
+<!-- ------------------------ -->
+## Lab Overview 
```

Comment:
Rename this section to "Overview" and also add "What You Will Build" section.

See https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html